src/main/resources/db/migration/V1_99_3__changelog_insert_data.sql

```diff
@@ -0,0 +1,13 @@
+insert into changelog (referral_id, changelog_id, topic, old_value, new_value, reason_for_change, changed_at, changed_by_id)
```

Comment:
do we need this?   I don't think we want every referral to have a migration entry on the changelog screen do we?

Reply:
No you are right we don't, wasn't sure what this was doing tbh - can remove it

Reply:
Cheers for that, makes a lot more sense now 👍 @Froom2 tagging you too so you have that as reference